easybuild/easyblocks/generic/pythonpackage.py

```diff
@@ -410,6 +411,17 @@ def prepare_python(self):
             # set Python lib directories
             self.set_pylibdirs()
 
+    def _should_unpack_source(self):
+        """Determine whether we need to unpack the source(s)"""
+        unpack_sources = self.cfg.get('unpack_sources')
+        if unpack_sources is None:
+            src = self.src
+            if not isinstance(src, string_type):
+                src = src[0]['path']
+            _, ext = os.path.splitext(src)
+            unpack_sources = ext.lower() == '.whl'
```

Comment:
This should be `!=`, no? Unpack when we're *not* using a `.whl` as source?

Also, why not handle this on the framework side instead, similar to https://github.com/easybuilders/easybuild-framework/pull/3545 ?

Reply:
Ouch, of course! How did this work then...?
I wouldn't mind, but am afraid, that this causes confusion and while we could unpack whl-files in general, changing the framework would disable that completely even if we wanted to unpack those in the future.

Reply:
Fixed the comparison